doc/source/cluster/kubernetes/user-guides/pod-command.md

```diff
@@ -63,91 +113,3 @@ Currently, for timing (1), we can set the container's `Command` and `Args` in Ra
     # Args:
     #    echo 123  456  && ulimit -n 65536; ray start --head  --dashboard-host=0.0.0.0  --num-cpus=1  --block  --metrics-export-port=8080  --memory=2147483648
     ```
-
-
-## Timing 2: After `ray start` (RayCluster is ready)
```

Comment:
I had to think a bit to remember that the reason we don't need this section anymore is because we can use Part 1 to just directly add post-start commands after the KUBERAY_GEN_RAY_START_COMMAND.  

I think it would help to make this really obvious and explicit in the doc for part 1, maybe just with a quick sentence or example. Or maybe just put a post start command in the existing example.

Reply:
This is an over-explanation for me. I will update it only if you feel strongly about its necessity.